code/game/objects/items/fireaxe.dm

```diff
@@ -85,3 +85,4 @@
 	demolition_mod = 2
 	tool_behaviour = TOOL_CROWBAR
 	toolspeed = 1
+	usesound = 'sound/items/crowbar.ogg'
```

Comment:
why not have this on parent?

Reply:
As far as I'm aware, usesound is associated with tool_behaviour which the standard fireaxe doesn't use.